packages/cli/src/oclif/commands/migrate.js

```diff
@@ -15,6 +15,7 @@ class MigrateCommand extends BaseCommand {
     const user = this.flags.user;
     const fromVersion = this.args.fromVersion;
     const toVersion = this.args.toVersion;
+    const email = this.args.email;
```

Comment:
This won't quite work. If you look below at `MigrateCommand.flags`, we only define the `user` flag. I think if you provide `--email`, the CLI will error at you.

Because the name `email` vs `user` doesn't matter, I'd keep the existing flag name and just save it as `email` in the request body. That way the flags don't change for our users, but we send the right data to the API. Does that make sense?

Reply:
Yep, makes sense, good catch!

> Because the name email vs user doesn't matter

Can you help me locate this for my own learning? I'm tracing where the endpoint builds the payload and find it set [here](https://github.com/zapier/zapier/blob/986861aa44dbbed208189c6f5b630e78828b4d9a/web/backend/zapier/migration_fsm/statedb.py#L63) but not sure where it will end up being used.

Reply:
https://cdn.zappy.app/eb7c153daa90542ec50c964bdc903dcb.png

nvm! I see now that user _is_ email in this system!